src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -379,8 +379,19 @@ public static function getProfile($uid)
 			'owner' => $contact['url'],
 			'publicKeyPem' => $user['pubkey']];
 		$data['endpoints'] = ['sharedInbox' => DI::baseUrl() . '/inbox'];
-		$data['icon'] = ['type' => 'Image',
-			'url' => $contact['photo']];
+		$data['icon'] = ['type' => 'Image', 'url' => $contact['photo']];
+
+		$resourceid = Photo::ridFromURI($contact['photo']);
+		if (!empty($resourceid)) {
+			$photo = Photo::selectFirst(['type'], ["resource-id" => $resourceid]);
+			if (!empty($photo['type'])) {
+				$data['icon']['mediaType'] = $photo['type'];
+			}
+		}
```

Comment:
Why don't we do the same for the header image?

Reply:
Okay.